voice/bxml/callbacks/disconnect.md

```diff
@@ -1,6 +1,19 @@
 {% method %}
 ## Disconnect Event
-The Disconnect event is fired when a call ends, for any reason.
+
+The Disconnect event is fired when a call ends, for any reason. The `cause` for a disconnect event on a call can be:
+- `hangup`: there was no more BXML to execute in it; it indicates that the call ended normally.
```

Comment:
Some places you say "do something _on_ the call", others "_in_ the call". Do you know what's best?
In this case particularly I think you can just remove "in it".